fluent/protocol/transport.go

```diff
@@ -112,16 +116,47 @@ type EntryExt struct {
 
 type EntryList []EntryExt
 
+func (el *EntryList) UnmarshalPacked(bits []byte) ([]byte, error) {
```

Comment:
Adding this allowed the `"Correctly encodes the entries into a bytestream` test case to work